docs/send-data/opentelemetry-collector/remote-management/source-templates/postgresql/changelog.md

```diff
@@ -0,0 +1,9 @@
+---
+id: changelog
+title: Changelog
+description: Changelog for PostgreSQL source template for OpenTelemetry.
+---
+
+<head>
+  <meta name="robots" content="noindex" />
+</head>
```

Comment:
@sumoanema I've added this placeholder changelog doc

Reply:
Have added the changelog in this file. Thanks